Tests/TuistKitTests/Generator/ConfigGeneratorTests.swift

```diff
@@ -84,55 +84,81 @@ final class ConfigGeneratorTests: XCTestCase {
             XCTAssertEqual(config?.buildSettings["CODE_SIGN_ENTITLEMENTS"] as? String, "$(SRCROOT)/Test.entitlements")
             XCTAssertEqual(config?.buildSettings["SWIFT_VERSION"] as? String, Constants.swiftVersion)
 
-            let xcconfig: PBXFileReference? = config?.baseConfiguration
-            XCTAssertEqual(xcconfig?.path, "\(config!.name.lowercased()).xcconfig")
+//            let xcconfig: PBXFileReference? = config?.baseConfiguration
+//            XCTAssertEqual(xcconfig?.path, "\(config!.name.lowercased()).xcconfig")
```

Comment:
xcconfig was removed as an option from the Target, this was supported before - should it be added in to support people who are using xconfig per target?

Reply:
I don't think the Xcode UI even lets you do this (unless I missed it), removing it would keep things consistent with what can be defined manually via Xcode :)

Reply:
I thought that's what this UI was for:

![09f999c6-af90-4903-937e-8f53873bee1d](https://user-images.githubusercontent.com/1382565/53161224-776b6d00-35c1-11e9-9d5a-9293c7171638.png)

Reply:
Ah of course! Completely missed this

Reply:
I'd keep the support for it. It's something that developers might be using so I'd better contemplate it in the new API design.

Reply:
the latest update restores it :)

<img width="675" alt="screenshot 2019-02-25 at 21 34 54" src="https://user-images.githubusercontent.com/11914919/53370273-3a96d180-3945-11e9-99e8-1929c52d05b1.png">

Reply:
That's right, I made sure I went back to add this in as it does seem like a pretty useful feature. The same behaviour described in the pull request description is still implemented too.